### ISSUE TITLE
Can't build nintengo.wasm,go version is 1.14

### ISSUE THREAD

**Issue by chenxun450** [nwidger/nintengo#21]
My owesome guy,I can't build it。
How should I deal this problem
E:\gopath\pkg\mod\github.com\nwidger\nintengo@v0.0.0-20190228002135-f173d610a4ac\nes\audio_js.go:34:9: undefined: js.TypedArrayOf
E:\gopath\pkg\mod\github.com\nwidger\nintengo@v0.0.0-20190228002135-f173d610a4ac\nes\video_js.go:207:10: undefined: js.TypedArrayOf

**Comment by nwidger** [comment]
@chenxun450 Yes, nintengo doesn't support Go 1.14 for WASM.  The API for the `syscall/js` package has changed since I initially added WASM support to nintengo and I haven't had the time to update to the new API.  I'd welcome any PR's that fix this situation, though!
